AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.ts

```diff
@@ -43,6 +43,7 @@ export class AllocateHearingsComponent implements OnInit {
     customIconTransform: Transform = { rotate: 45 };
     private filterSize = 20;
     dropDownUserLabelAllocateTo = 'Allocate to';
+    readonly HEARING_HAVE_BEEN_UPDATED = 'Hearings have been updated.';
```

Comment:
we have a Constants file (in src/app/common/constants.ts) which we use for messages. Could we store it in there instead?

with an entry something like:

`AllocateHearings: {
        Confirmed: 'Hearings have been updated.'
    }`
    
but good idea nonetheless though, moving it into a constant :)

Reply:
Yes, I was looking for such a file.
Confimation message moved to constants file now.